bentoml/yatai/client/bento_repository_api.py

```diff
@@ -344,62 +346,73 @@ def list(
             raise BentoMLException(f'{error_code}:{error_message}')
         return result.bentos
 
-    def delete(self, bento):
+    def delete(
```

Comment:
could we make the client back-wards compatible by also providing the `yatai_client.repository.delete('Name:Version')` syntax?  Making the first parameter `bento_tag`, without changing the grpc API under the hood

Reply:
that way you won't need to change the unit/integration test code